julia_src/http.jl

```diff
@@ -64,50 +64,76 @@ function reopt(req::HTTP.Request)
 			)
 		)
 	end
-    @info "Starting REopt..."
+	@info "Starting REopt..."
     error_response = Dict()
     results = Dict()
 	inputs_with_defaults_set_in_julia = Dict()
-    try
+	model_inputs = nothing
+	# Catch handled/unhandled exceptions in data pre-processing, JuMP setup
```

Comment:
Since we're separating this out, maybe could make the error message more specific, saying the error occurred in input preprocessing?

Reply:
Yup, updated